### PR TITLE
apache config files

### PR DESCRIPTION
**Pull request by kamfonik**
Config files for using apache server.

### CODE REVIEW

**Comment by kamfonik** [comment]
@knikolla @jeremyfreudberg while everyone is thinking about this repo, can I get a +1 for this httpd config?

File: httpd/apache.conf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Listen 5001 https
+
+<VirtualHost *:5001>
+  SSLEngine On
+  SSLCertificateFile /PATH/TO/CERT.crt
+  SSLCertificateKeyFile /PATH/TO/KEY.key
+
+  WSGIPassAuthorization On
+  WSGIChunkedRequest On
+  WSGIDaemonProcess webtool user=apache group=apache threads=5 home=/var/www/setpass/
```
Comment (jeremyfreudberg) [inline httpd/apache.conf:13]: Looks good to me, but we could discuss the number of threads (or processes, they are separate options) further. Not really my area of expertise though

Comment (kamfonik) [reply httpd/apache.conf:13]: That bit is copy-paste from a k2k script that I didn't change, so certainly open to discussion/critique.  I'll read up on the options when I get a chance.

**Review by knikolla — APPROVED** [review]
I assume we're already using this and it works. so +1

**Comment by kamfonik** [comment]
Yes, we're using this in production.

**Pull request merged**